tox.ini

```diff
@@ -49,26 +45,16 @@ whitelist_externals = {[testenv]whitelist_externals}
 commands =
     # We run the upgrade tests 1st because assuming a fast internet connection,
     # (to download the pre_built image of pulp 3.0.0), they take less long.
-    molecule lint -s default-upgrade
-    molecule converge -s default-upgrade # calls: dependency, create and prepare
-    molecule syntax -s default-upgrade
+    {env:TRAVIS_WAIT:} molecule test -s default-upgrade
```

Comment:
Moving the individual tasks to be run to molecule.yml yields much more elegant code, and I overall approve of this approach.

The only downside would be that we could be stuck waiting until Travis completely times out (60 mins IIRC) on test stages other than the long-running-with-no-output idempotence test. However, empirically that has never happened yet (or I cannot recall it), so I agree with this approach.

I do think we need to add comment about why we have the TRAVIS_WAIT (for the idempotence test.)

Reply:
Or rather, move the comment to the first instance of TRAVIS_WAIT in the file.